src/prompt_toolkit/shortcuts/dialogs.py

```diff
@@ -109,6 +109,7 @@ def input_dialog(
     validator: Optional[Validator] = None,
     password: FilterOrBool = False,
     style: Optional[BaseStyle] = None,
+    default_text: str = "",
```

Comment:
Can you rename the parameter to ``default`` here?
For consistency with ``PromptSession``: https://github.com/prompt-toolkit/python-prompt-toolkit/blob/master/src/prompt_toolkit/shortcuts/prompt.py#L899

I know for the ``radiolist_dialog``, it's ``default_values``, but I think there it can be more than one.

Reply:
Ok